doc/source/reference/arrays.dtypes.rst

```diff
@@ -225,16 +231,18 @@ Array-protocol type strings (see :ref:`arrays.interface`)
    supported kinds are
 
    ================   ========================
-   ``'b'``            boolean
+   ``'?'``            boolean
+   ``'b'``            (signed) byte
+   ``'B'``            unsigned byte
```

Comment:
If we really want completeness here, then we support (most of?) the stuct pack codes too (https://docs.python.org/2/library/struct.html#format-characters). Certainly, `bBhHiIlLqQ` are all accepted